rest.go

```diff
@@ -161,14 +169,14 @@ func httpEngine() *gin.Engine {
 	rg.GET("builds", getBuilds)
 
 	rg.POST("benchmarks", addBenchmark)
-	rg.GET("benchmarks/:component/:category", getBenchmarks)
+	rg.GET("benchmarks/:component/:category/:subCategory", getBenchmarks)
```

Comment:
Please keep the existing API and create a new router group "/api/v2" for new endpoints.

API v1 is already used by external projects.